space-template.yaml

```diff
@@ -0,0 +1,166 @@
+---
+# These define the potential field types that ca be used in the fields of work
+# item types.
+field_types:
+  - &string string
+  - &translatable_string translatable_string
+  - &list list
+  - &timestamp timestamp
+  - &user user
+
+# A map of common strings that can be used to denote states of a work item
+common_work_item_states: &common_work_item_states
+  &state_new new:
+      en_US: "new"
```

Comment:
@maxandersen @michaelvirgil  I have removed the i18n in this commit: 4c6c70f350bd665fad50649e9e34f9d5437ecdc8